source/_posts/2023-10-01-php-core-roundup-17.md

```diff
@@ -0,0 +1,330 @@
+
+<br>
+Hundreds of awesome PHP contributors put their efforts into improvements to the PHP code base, documentation, and the php.net website. Here is a summary of some changes made by the people behind PHP. Things marked with 💜 are done by the PHP Foundation team.
+
+## PHP 8.3 Release Page
+
+A [pull-request](https://github.com/php/web-php/pull/807) for the upcoming PHP 8.3 release page on php.net is in the progress, and you can help with that!
+
+This is a continuation of [a good tradition](https://externals.io/message/112026) started by Roman Pronskiy, Alexander Makarov, and Svetlana Belozerova.
+
+Check out how these pages looked like for [PHP 8.0](https://www.php.net/releases/8.0/en.php), [PHP 8.1](https://www.php.net/releases/8.1/en.php), and [PHP 8.2](https://www.php.net/releases/8.2/en.php).
+
+## Early-developments for PHP 8.4
+
+Although PHP 8.3 is still being ironed out, there are some discussions and even an RFC currently being voted for proposed changes in PHP 8.4 (scheduled for the end of 2024).
+
+### Declined: [Support optional suffix parameter in tempnam](https://wiki.php.net/rfc/tempnam-suffix-v2) by Athos Ribeiro
+
+RFC proposes to add a new optional suffix parameter to the `tempnam()` function.
+
+A suffix could provide even more semantic value or context for a user inspecting the generated files, and, in specific situations, could even provide more context for software processing such files. Right now, users can only add a prefix.
+
+### In Voting: [Increasing the default BCrypt cost](https://wiki.php.net/rfc/bcrypt_cost_2023) by Tim Düsterhus
+
+RFC proposes to increase default BCrypt cost, which denotes the algorithmic cost that should be used, from 10 to 11 (doubling the time) or 12 (quadrupling the time). The [RFC](https://wiki.php.net/rfc/bcrypt_cost_2023) and the relevant [mailing list thread](https://externals.io/message/121004) mention several benchmarks showing the execution time for various cost levels on different CPUs.
+
+### Under Discussion: [DOM HTML5 parsing and serialization](https://wiki.php.net/rfc/domdocument_html5_parser) by Niels Dossche
+
+RFC proposes to add two new classes: `DOM\HTMLDocument` and `DOM\XMLDocument` to the dom extension. Furthermore, existing dom classes in the global namespace get an alias in the new DOM namespace. The `HTMLDocument` class will add support for HTML5 document parsing and serializing. The `XMLDocument` class serves as a modern alternative to `\DOMDocument`, which is retained for compatibility. These new classes also provide a more misuse-resistant API for loading documents.
+
+### Under Discussion: [XML_OPTION_PARSE_HUGE](https://wiki.php.net/rfc/xml_option_parse_huge) by Niels Dossche
+
+RFC proposes to add a new `XmlParser` option to allow large documents to be parsed.
```

Comment:
```suggestion
RFC proposes to add a new `XmlParser` option to allow large documents to be parsed in an event-driven way (SAX).  
```

Reply:
I believe this phrasing can be misunderstood: To me it sounds like this adds a SAX parser itself, but it just allows using the SAX parser for large documents.

Perhaps:

> RFC proposes to add a new option to the event-driven (SAX) `XmlParser` that allows it to parse large documents.

/cc @nielsdos